node/src/main/resources/sqlite/node/migration/V4__new_peer_table.sql

```diff
@@ -0,0 +1,3 @@
+drop table "peers";
```

Comment:
Cannot know the `service_bytes` value for existing peers so dropping that table seems the only option.